deployment/ecr/distributed-load-testing-on-aws-load-tester/load-test.sh

```diff
@@ -33,6 +33,22 @@ if [ "$TEST_TYPE" != "simple" ]; then
     fi
 
     sed -i -e "s|$TEST_ID.jmx|$JMETER_SCRIPT|g" test.json
+
+    # copy bundled plugin jars to jmeter extension folder to make them available to jmeter
+    BUNDLED_PLUGIN_DIR=`find $PWD -type d -name "plugins"`
```

Comment:
Can you add to check only `$PWD`? For example, this would return multiple directories if there are multiple `plugins` directories in the zip file somehow.

```shell
% find $PWD -type d -name "plugins"
/test/inner/plugins
/test/plugins
```

In that case, I believe `cp` command would fail.

Reply:
Thanks for that and your patience.

This is surprisingly hard, as the uploaded zip file may have an arbitrary folder structure. For example when zipping a folder with configurations, the zip file will contain a top level folder with unknown name. Therefore the `/plugins` folder may be nested actually.

Ultimately I used the same approach as for the jmx file - use the first occurence of a `plugins` directory found in the zip.

![image](https://user-images.githubusercontent.com/2890900/172073267-d9bf18f4-dc9d-4ede-bae1-0df06b5560c3.png)

Reply:
@beomseoklee can you take another look please?